packages/react-vapor/src/components/dropdownSearch/MultiSelectDropdownSearch/tests/MultiSelectInput.spec.tsx

```diff
@@ -23,48 +23,58 @@ describe('MultiSelectInput', () => {
             multiSelectInput = mount(<MultiselectInput {...curentProps} />, {attachTo: document.getElementById('App')});
         };
 
-        beforeEach(() => {
-            renderMultiSelectInput(props);
-        });

```

Comment:
@aiheon how come we're moving the `renderMultiSelectInput` into each test instead of a `beforeEach`?  Just curious on the change

Reply:
It's a new rule in eslint 8 :see_no_evil: 

We are still not set on if we applied it or not :slightly_smiling_face:  (I'm not sure if I want to make the change in ALL the test file since admin-ui is next but it's done in Plasma at least :see_no_evil: )

Reply:
Oh okay, cool TIL.  I assume something like not rendering in beforeAll/beforeEach?  Anyways, thanks for clarifying.